### PR TITLE
New NetBricks-dev setup

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
Checks
======

- [x] Needs https://github.com/williamofockham/libmoon/pull/6 until that PR is merged. 
- [x] Needs https://github.com/williamofockham/MoonGen/pull/23 until that PR is merged.
- [x] Needs https://github.com/williamofockham/utils/pull/37 (most importantly) until that PR is merged.

-----------------

* cleans-up working w/ NetBricks, akin to our internal og work
* updates README accordingly
* uses -n as part of tcpdump know to remove host names, and updates the test exepcteds accordingly
* updates travis config accordingly

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
I'm down for README fixums and such too.

**Comment by zeeshanlakhani** [comment]
@scrollins, 

1) Netbricks will still be mounted into the OG one, but, we no longer will have OG building-off the Netbricks container. The utils/sandbox container is now the main dev-container env. 

2) I updated all the ones we run accordingly as part of the suite. The one you're working on will need an update then. But, the key is adding `-n` as part of the tcpdump output: https://github.com/williamofockham/NetBricks/pull/55/files#diff-a2c563693c1684d44403a41eef9c1779R16. It makes sure we don't get any weird renaming.

**Comment by scrollins** [comment]
why don't all of the tests have to be updated? we do tcpdumps on all of them

**Comment by scrollins** [comment]
+1 LGTM

File: README.md [hunk 1]
```diff
@@ -4,104 +4,168 @@
...
+   ```
+
+4. Symlink the Vagrantfile into the parent directory.
+   ```shell
+   host$ ln -s utils/Vagrantfile
+   ```
+
+4. Boot the VM:
+   ```shell
+   host$ vagrant up
```
Comment (pauljamescleary) [inline README.md]: following these directions, I encountered:

```
There are errors in the configuration of this machine. Please fix
the following errors and try again:

shell provisioner:
* `path` for shell provisioner does not exist on the host system: /Users/pclear200/dev/github/williamofockham/vm-kernel-upgrade.sh
* `path` for shell provisioner does not exist on the host system: /Users/pclear200/dev/github/williamofockham/vm-setup.sh
```

Comment (pauljamescleary) [reply README.md]: Also, the name collides with our internal dev environment...

```
A VirtualBox machine with the name 'ubuntu-xenial-williamofockham' already exists.
Please use another name or delete the machine with the existing
name, and try again.
```

Should we use a different name for this virtual box vs. internal?

Comment (zeeshanlakhani) [reply README.md]: Agreed on the name @pauljamescleary. The error you saw is due to needing the PRs in https://github.com/williamofockham/utils/pull/37, https://github.com/williamofockham/libmoon/pull/6, and https://github.com/williamofockham/MoonGen/pull/23.

**Pull request merged**